recipes/pyomo/meta.yaml

```diff
@@ -0,0 +1,170 @@
+{% set version = "4.3.11377" %}
+
+package:
+  name: pyomo
+  version: {{ version }}
+
+source:
+  fn: Pyomo-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/12/8e/edae55601011b4356f26e3d3346e0032a67a89cd2621ff893e0a623d68ab/Pyomo-4.3.11377.tar.gz
+  #url: https://pypi.python.org/packages/source/P/Pyomo/Pyomo-{{ version }}.tar.gz
+  md5: 6e27c7b5c1398bc56156ba9d56b43679
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  entry_points:
+    - runbenders=pyomo.pysp.benders:Benders_main
+    - evaluate_xhat=pyomo.pysp.evaluate_xhat:EvaluateXhat_main
+    - runph=pyomo.pysp.phinit:PH_main
+    - runef=pyomo.pysp.ef_writer_script:main
+    - pysp2smps=pyomo.pysp.smps.pysp2smps:main
+    - phsolverserver=pyomo.pysp.phsolverserver:main
+    - scenariotreeserver=pyomo.pysp.scenariotree.server_pyro:main
+    - computeconf=pyomo.pysp.computeconf:main
+    - results_schema=pyomo.scripting.commands:results_schema
+    - pyro_mip_server=pyomo.scripting.pyro_mip_server:main
+    - test.pyomo=pyomo.scripting.runtests:runPyomoTests
+    - pyomo=pyomo.scripting.pyomo_main:main
+    - pyomo_ns=pyomo.scripting.commands:pyomo_ns
+    - pyomo_nsc=pyomo.scripting.commands:pyomo_nsc
+    - kill_pyro_mip_servers=pyomo.scripting.commands:kill_pyro_mip_servers
+    - launch_pyro_mip_servers=pyomo.scripting.commands:launch_pyro_mip_servers
+    - readsol=pyomo.scripting.commands:readsol
+    - ossolverservice=pyomo.scripting.commands:OSSolverService
+    - pyomo_python=pyomo.scripting.commands:pyomo_python
+    - pyomo_old=pyomo.scripting.pyomo_command:main
+    - get_pyomo_extras=scripts.get_pyomo_extras:main
```

Comment:
There are a few auxiliary scripts that are distributed with Pyomo but which
aren't part of the source tree.  I agree that this one shouldn't be treated
as providing an entry point.  I'll remove it shortly.

On Tue, Apr 26, 2016 at 9:14 PM, jakirkham <notifications@github.com> wrote:

> In recipes/pyomo/meta.yaml
> <https://github.com/conda-forge/staged-recipes/pull/449#discussion_r61190706>
> :
>
> > +    - phsolverserver=pyomo.pysp.phsolverserver:main
> > +    - scenariotreeserver=pyomo.pysp.scenariotree.server_pyro:main
> > +    - computeconf=pyomo.pysp.computeconf:main
> > +    - results_schema=pyomo.scripting.commands:results_schema
> > +    - pyro_mip_server=pyomo.scripting.pyro_mip_server:main
> > +    - test.pyomo=pyomo.scripting.runtests:runPyomoTests
> > +    - pyomo=pyomo.scripting.pyomo_main:main
> > +    - pyomo_ns=pyomo.scripting.commands:pyomo_ns
> > +    - pyomo_nsc=pyomo.scripting.commands:pyomo_nsc
> > +    - kill_pyro_mip_servers=pyomo.scripting.commands:kill_pyro_mip_servers
> > +    - launch_pyro_mip_servers=pyomo.scripting.commands:launch_pyro_mip_servers
> > +    - readsol=pyomo.scripting.commands:readsol
> > +    - ossolverservice=pyomo.scripting.commands:OSSolverService
> > +    - pyomo_python=pyomo.scripting.commands:pyomo_python
> > +    - pyomo_old=pyomo.scripting.pyomo_command:main
> > +    - get_pyomo_extras=scripts.get_pyomo_extras:main
>
> So, this one seems a little odd. Is there suppose to be a pyomo before?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/conda-forge/staged-recipes/pull/449/files/62899347856be9cfe0db02879d5935a43c0455cd#r61190706>
>